generators/app/templates/graphQL/test/app.spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { hostRedis } = require('../config').common.redisCache;
+
+const fs = require('fs'),
+  models = require('../app/models'),
+  path = require('path'),
+  RedisClient = require('redis');
+
+const tables = Object.values(models.sequelize.models);
+
+const truncateTable = model =>
+  model.destroy({ truncate: true, cascade: true, force: true, restartIdentity: true });
+
+const truncateDatabase = () => Promise.all(tables.map(truncateTable));
+
+const flushRedis = redisClient =>
+  new Promise(resolve => {
+    redisClient.sendCommand('FLUSHALL');
+    redisClient.quit();
+    resolve();
+  });
+
+beforeEach(done => {
+  Promise.all([flushRedis(RedisClient.createClient({ hostRedis })), truncateDatabase()]).then(() => done());
+});
+
+// including all test files
+const normalizedPath = path.join(__dirname, '.');
+
+const requireAllTestFiles = pathToSearch => {
+  fs.readdirSync(pathToSearch).forEach(file => {
+    if (fs.lstatSync(`${pathToSearch}/${file}`).isDirectory()) {
+      requireAllTestFiles(`${pathToSearch}/${file}`);
+    } else {
+      require(`${pathToSearch}/${file}`);
+    }
+  });
+};
+
+requireAllTestFiles(normalizedPath);
```

Comment:
I would try to avoid using this script to require all tests.

In Express.js template, we are using `"setupFilesAfterEnv"` config for jest.

- Here the file https://github.com/Wolox/express-js-bootstrap/blob/master/generators/app/templates/test/setup.js
- Link to package.json config for Jest https://github.com/Wolox/express-js-bootstrap/blob/b287c957066628dc44e69f7b5a62df852c17cd54/generators/app/templates/package.json#L64-L66

Reply:
Hello, I already added it, but I don't know if I understood you, you could check it and tell me if I understood well what had to be done.